### DISCUSSION TITLE
copilot is amazing!

### DISCUSSION THREAD

**Discussion by madmaxaus** [community/community#7330] [Copilot Conversations]
i really love it. Saves me so much time (no more copying basic functions from stack overflow)

**Comment by onyet** [comment]
this really surprised me. how can it understand what I'm going to want to do. 
This is very time-saving and quite surprising. ❤️

**Reply by LuBreGit** [reply]
Agreed, I'm testing it now and it looks fantastic

**Reply by Shakedkod** [reply]
Indeed

**Comment by sagive** [comment]
omg! it's awesome... Saved me an hour in the first hour. Amazing job guys - you got a six-pack waiting here any day any time...

**Comment by LuBreGit** [comment]
It's also a great "learning" tool, I can't deny that. A lot of tips and stuff to take inspiration from.

**Comment by YusufSuleman** [comment]
Me too.

**Comment by jfrei** [comment]
I just wanted to continue the love for Copilot...

Occasionally I find it predicting exactly what I intend to code without writing any hints or any seed code, and it's just amazing to see that.

If it needs context I generally try to throw in some comments before a block of code and that usually provides good guidance for it to start throwing out more applicable suggestions.

Apart from using Copilot as a code generation/completion tool, I find that I too am learning different methodologies based on its suggestions, which is quite refreshing. It gets me out of doing the same stagnant habits that I can fall into and shows me great alternatives that are much cleaner and a bit more modern looking.

Super great job team :)
